src/components/internal/OverlayTrigger.tsx

```diff
@@ -76,6 +78,7 @@ export function OverlayTrigger(props: OverlayTriggerProps) {
           disabled={disabled}
           tooltip={tooltip}
           onClick={noop}
+          forceFocusStyles={showActiveBorder && state.isOpen}
```

Comment:
I kept the `showActiveBorder` flag because we don't want to apply an active border for all cases

Reply:
**question** When using the ButtonModal should `showActiveBorder` something we should support for all possible triggers? That'd include IconButton and AvatarButton.